Bottom: c0f7ba20f514b5e86a37fe0455707ca40b3fd0f7
Top:    b8bbdb8ed1a36de7f3a74d97fffa62269479bd1b
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-07-27 17:19:23 +0800

Refresh of tun-use-build_skb-for-small

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index c8e8a32..de1a2ef 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1341,7 +1341,7 @@ static ssize_t tun_get_user(struct tun_struct *tun, struct tun_file *tfile,
 		skb = tun_alloc_skb(tfile, align, copylen, linear, noblock);
 		if (IS_ERR(skb)) {
 			if (PTR_ERR(skb) != -EAGAIN)
-			this_cpu_inc(tun->pcpu_stats->rx_dropped);
+				this_cpu_inc(tun->pcpu_stats->rx_dropped);
 			return PTR_ERR(skb);
 		}
